pkg/upgrade/upgrades/role_members_ids_migration_test.go

```diff
@@ -36,10 +36,10 @@ func TestRoleMembersIDMigration10Users(t *testing.T) {
 	runTestRoleMembersIDMigration(t, 10)
 }
 
-func TestRoleMembersIDMigration1500Users(t *testing.T) {
+func TestRoleMembersIDMigration1000Users(t *testing.T) {
 	skip.UnderRace(t)
 	skip.UnderStress(t)
-	runTestRoleMembersIDMigration(t, 1500)
+	runTestRoleMembersIDMigration(t, 1000)
```

Comment:
Hmm I specifically picked a number > 1000 to test that the migration would work with multiple batches:
https://github.com/cockroachdb/cockroach/blob/2f811420a1492ca547659003f193233341b7fd87/pkg/upgrade/upgrades/role_members_ids_migration.go#L78-L84

Would 1100 work? Or even 1001?

Reply:
that sounds good